modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMetaStorage.java

```diff
@@ -54,6 +54,11 @@ public class RocksDbMetaStorage {
      */
     public static final byte[] INDEX_ROW_ID_PREFIX = {2};
 
+    /**
+     * Prefix to store lease start time. Key format is {@code [prefix, tableId, partitionId]} in BE.
+     */
+    static final byte[] LEASE_PREFIX = {3};
```

Comment:
Do we clean it on partition/table destruction? I think you forgot it

Reply:
fixed